packages/react-components/src/button/button.tsx

```diff
@@ -570,6 +691,8 @@ export const Button: FunctionComponent<ButtonProps> = React.forwardRef(
         ref={ref}
         color={color}
         isIcon={!!icon}
+        leftIcon={!!leftIcon}
```

Comment:
@Pratosh22 We already have `leftIcon` and `rightIcon` props for our button right ? Wont this get overridden when we spread {...rest} below ?